stdpopsim/cli.py

```diff
@@ -708,20 +721,40 @@ def run_simulation(args):
                     "A DFE annotation has been assigned without a DFE. "
                     "Please specify a DFE."
                 )
+            elif args.dfe_bed_file is not None:
+                exit(
+                    "A DFE bed file has been assigned without a DFE. "
+                    "Please specify a DFE."
+                )
         else:
             if args.dfe_interval is not None:
                 if args.dfe_annotation is not None:
                     exit(
                         "A DFE annotation and a DFE interval have been "
                         "selected. Please only use one."
                     )
+                if args.dfe_bed_file is not None:
+                    exit(
+                        "A DFE bed file and a DFE interval have been "
+                        "selected. Please only use one."
+                    )
                 left, right = args.dfe_interval.split(",")
                 intervals = np.array([[int(left), int(right)]])
                 intervals_summary_str = f"[{left}, {right})"
             elif args.dfe_annotation is not None:
+                if args.dfe_bed_file is not None:
+                    exit(
+                        "A DFE bed file and a DFE annotation have been "
+                        "selected. Please only use one."
+                    )
                 annot = species.get_annotations(args.dfe_annotation)
                 intervals = annot.get_chromosome_annotations(args.chromosome)
                 intervals_summary_str = f"{annot.id} elements on {args.chromosome}"
+            elif args.dfe_bed_file is not None:
+                intervals = np.loadtxt(args.dfe_bed_file, usecols=[1, 2], dtype="int")
```

Comment:
did you check if the error it gives for a malformed BED file (eg accidentally calling it on something totally different) is interpretable?

Reply:
No, doing it now

Reply:
ok, it is something like:
OSError: "bla.bed" not found.
so interpretable, do we need to test this?

Reply:
I was thinking a file that exists but is the wrong format - but we can open a new issue for this if we want.